src/Orleans/Runtime/GrainInterfaceMap.cs

```diff
@@ -83,6 +85,16 @@ internal IList<int> SupportedGrainTypes
             get { return implementationIndex.Keys.ToList(); }
         }
 
+        internal IList<GrainClassData> SupportedGrainClassData
+        {
+            get { return implementationIndex.Values.ToList(); }
```

Comment:
They are not used in our code. I just kept them for debugging purpose. I could remove them, so no one can call them later in "real" code...